apps/webapp/src/state/staking.test.ts

```diff
@@ -0,0 +1,96 @@
+import { beforeEach, describe, expect, it, vi } from 'vitest';
```

Comment:
This file is pretty much moved from `use-validator-infos.test.ts`, but with some changes to account for using Zustand instead of context